assets/locales/service.en.toml

```diff
@@ -29,10 +29,6 @@ one = "Data includes"
 description = "how many people live there"
```

Comment:
We should try to keep the welsh .toml file up to date too with the same keys (copy-paste job at this point), they will just contain the English values for each key initially)